client/landing/gutenboarding/onboarding-block/colors.scss

```diff
@@ -0,0 +1,95 @@
+.gutenboarding-color-coded {
```

Comment:
This file is now in `master`, time to rebase. :-)

Reply:
Rebased and no longer uses `.gutenboarding-color-coded.`